src/pages/SplitPosition/Form.tsx

```diff
@@ -172,11 +172,8 @@ export const Form = ({
             collateral
           )
         } else if (splitFromPosition && position) {
-          const {
-            collateralToken: { id: collateral },
-            collection: { id: collectionId },
-          } = position
-
+          collateral = position.collateralToken?.id
+          const collectionId = position.collection?.id
           await splitPosition(collateral, collectionId, conditionId, partition, amount)
```

Comment:
I could be missing something here, but to me seems that the problem was caused by the use of  `collateral` instead of `collateralToken` and `collectionId` instead of `collection`.  I prefer to change that if for `splitFromPosition && position && position.collateralToken && position.collection` and make the destructuring as it was done before but changing the call to `splitPosition`.



This could also be fixing #289, can you check that?

Reply:
Added `collateralFromSplit` to the PR.